01_fib/lexual/main.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"math"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+// print out n fibonacci numbers, use negafibonacci if n < 0
```

Comment:
I know it isn't normally required to include documentation for non exported function, but for future reference when you are including documentation for an exported method or function, it is general convention to start with the name of the function / method. Eg.
```go
// fib prints out n fibonacci numbers, use negafibonacci if n < 0
func fib(n int) {
  ...
}
```